webapp/src/components/backstage/automation/input_keywords.tsx

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useState} from 'react';
+import styled from 'styled-components';
+
+import {StylesConfig} from 'react-select';
+
+import CreatableSelect from 'react-select/creatable';
+
+import {AutomationHeader, AutomationTitle, SelectorWrapper} from 'src/components/backstage/automation/styles';
+import {Toggle} from 'src/components/backstage/automation/toggle';
+
+interface Props {
+    enabled: boolean;
+    textOnToggle: string;
+    onToggle: () => void;
+    placeholderText: string;
+    keywords: string[];
+    onKeywordsChange: (keywords: string[]) => void;
+}
+
+const selectComponents = {
+    DropdownIndicator: null,
+    ClearIndicator: null,
+};
+
+const createOption = (label: string) => ({
+    label,
+    value: label,
+});
+
+export const InputKeywords = (props: Props) => {
+    const [inputValue, setInputValue] = useState('');
+    const [values, setValues] = useState(props.keywords.map(createOption));
+
+    const handleChange = (value: any) => {
+        let newValues: {label: string, value: string}[] = value;
+        if (!value) {
+            newValues = [];
+        }
+        setValues(newValues);
+        props.onKeywordsChange(newValues.map((item) => item.value));
+    };
+
+    const handleInputChange = (newInputValue: string) => {
+        setInputValue(newInputValue);
+    };
+
+    const handleInput = (isBlur: boolean, event: any) => {
```

Comment:
As before, is it possible to type `event` here? Maybe something like
```suggestion
    const handleInput = (isBlur: boolean, event: React.KeyboardEvent | React.FocusEvent) => {
```